modules/ai/module.php

```diff
@@ -165,6 +167,41 @@ public function __construct() {
 		add_action( 'elementor/element/common/_section_transform/after_section_end', [ $this, 'register_ai_hover_effect_control' ], 10, 1 );
 	}
 
+	public function handle_kit_install( $imported_data ) {
+		if ( 'success' !== $imported_data['status'] ) {
```

Comment:
```suggestion
		if ( isset( $imported_data['status'] ) && 'success' !== $imported_data['status'] ) {
```
Maybe, just to be super safe?